### PR TITLE
what is the distribution2.0

### PR DESCRIPTION
**Pull request by vinoo-igem**
A start to #178. An intro, with a bit of vision, while trying not to delve too deep into other things (for now). Written in "iGEM"'s voice, but happy to take input. 

I'll continue working on this, but I think starting the structure of where it lives and some of the content will be helpful.

I'd like to add additions like contents of the distribution (including specs), software/tooling, etc. And of course, make it all a bit nicer.

### CODE REVIEW

File: docs/introduction/distribution.rst [hunk 1]
```diff
@@ -0,0 +1,10 @@
...
+===========================
+
+iGEM is designing and building a new DNA distribution entirely from the ground up with support from the FreeGenes Project and the iGEM Engineering Committee. The focus of this and future distributions is in improving the quality of the parts and devices within, through curation and new interlaboratory studies run by the Engineering Committee. We hope these new changes will help iGEM teams push all facets of the design-build-test cycle for their projects further.
+
+The new iGEM distribution will be manufactured and distributed by the FreeGenes Project, with all physical material provided under the BioBrick Foundation’s Unilateral OpenMTA. Anyone will be able to request and use the iGEM distribution! 
+
+This also marks the first time that iGEM is collaboratively designing the distribution. We’ve worked closely with the iGEM Engineering Committee on the contents of this year’s distribution. This has also led to collaborative building and testing of new software infrastructure to support this endeavor. All of the associated material within this distribution, from the contents to the software, is available through this github repository!
```
Comment (noahsprent) [inline docs/introduction/distribution.rst:8]: *GitHub

**Review by noahsprent — COMMENTED** [review]
Thanks for writing this @vinoo-igem! It's a good summary of the major inputs into the new Distro. I may have a different view of this to you, but for me the key part of the new distribution is the fact that it is radically transparent and collaborative. Our efforts this year have really just been the beta test of the process so that far more people can contribute. I believe that the previous distribution was also curated, and delivered to iGEM teams for free, so we need to explain to people what is different this time around. I think a lot of people also won't know that much about the OpenMTA, it's certainly taken me a long time to understand it. If I haven't gotten that wrong, I would suggest beginning more on the lines of the last two paragraphs to give more of the 'philosophy' to begin with before going into the detail about things like Interlab and the OpenMTA.

**Comment by noahsprent** [comment]
May I suggest a review also from @plouisross? Who may be able to provide input on messaging from his perspective.

**Comment by noahsprent** [comment]
Oh by the way from a code point of view looks perfect to me, see here: https://igem-distribution.readthedocs.io/en/add-intro-doc/index.html

**Comment by vinoo-igem** [comment]
> Thanks for writing this @vinoo-igem! It's a good summary of the major inputs into the new Distro. I may have a different view of this to you, but for me the key part of the new distribution is the fact that it is radically transparent and collaborative. Our efforts this year have really just been the beta test of the process so that far more people can contribute. I believe that the previous distribution was also curated, and delivered to iGEM teams for free, so we need to explain to people what is different this time around. I think a lot of people also won't know that much about the OpenMTA, it's certainly taken me a long time to understand it. If I haven't gotten that wrong, I would suggest beginning more on the lines of the last two paragraphs to give more of the 'philosophy' to begin with before going into the detail about things like Interlab and the OpenMTA.

Thanks for the suggestion! I'll play around with the order. 

This is also a very good point, and something that I struggle with as I can imagine some folk have different takes on what is the most important change for the distribution. I do agree that it all circles back to its collaborative nature but I would like users to also understand the intention behind the final product.

I'll start working on fleshing out these other details as well. I would like to get something up soon though, even if it feels incomplete.

**Comment by noahsprent** [comment]
How about we just merge this for now, and open an issue to maybe play with the order? There's nothing wrong with it as it is, and better to have something than nothing?

**Review by noahsprent — CHANGES_REQUESTED** [review]
I think this gets across what we're trying to say, and we should just merge this now and we can change it later if we want. One minor change which I've only thought of looking at the commits here is that it would be good to have each sentence as it's own line for diffing purposes. There will only be a new paragraph if there is a double line break.

File: docs/introduction/distribution.rst (hunk 1 above)
Comment (plouisross) [reply docs/introduction/distribution.rst:8]: Following a revised version with some additional points,   Need to ensure accuracy.  Some further clarification required.
-------------------------

The iGEM Distribution was initiated to put a toolkit of genetic parts at the disposal of IGEM teams.  The goal of the program is to enable is to enable teams to devote more of their effort and attention to the design and invention of unique concepts and effective solutions and to reduce time spent searching for and developing the components on which achieving satisfactory results depends.    

The output of the distribution is a diverse range of elements such as plasmids, xxxx,  whose development is complex, involving rationalization of disparate naming conventions, translation between incompatible structures, modification of sequences 

What distinguishes this year’s distribution program from those of previous years and represents an important step forward is the scale of the collaborative effort undertaken to support the program and the introduction of a new software infrastructure designed to increase the efficiency of development. 

At the heart of the collaborative effort is the Engineering Committee whose members have devoted significant time and effort to ensuring the creation of useful and usable content.  However, the extent of the collaboration has not been limited to internal contributors, but encompasses external parties as well.   FreeGenes, Twist, and xxx are just a few of the organization that have contributed to and supported the distribution program. 

All of the material associated with this distribution, from the contents to the software, as well as supplemental information resides within the  GitHub repository (provide link here).

**Review by noahsprent — APPROVED** [review]

**Pull request merged**